Makefile

```diff
@@ -40,11 +40,14 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
 test-e2e: GO_TEST_FLAGS += -v
 test-e2e: test-unit
 
+PKG := ${PWD:${GOPATH}/%=%}
 update-codegen-crds:
-	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --domain openshift.io --apis-dir vendor/github.com/openshift/api
+	docker pull registry.svc.ci.openshift.org/ocp/4.2:crd-schema-gen
```

Comment:
is the pull necessary? Isn't there some option for docker run?

Reply:
You are probably right, I'll look